tritium-lib/build.gradle

```diff
@@ -1,6 +1,10 @@
 apply plugin: 'com.palantir.external-publish-jar'
 apply plugin: 'com.palantir.metric-schema'
 
+// work around metric-schema gradle 8 task dependency
+tasks.sourcesJar.dependsOn(generateMetrics)
+tasks.sourcesJar.dependsOn(compileMetricSchema)
```

Comment:
Created https://github.com/palantir/metric-schema/pull/1092 to remove need for this